tello_sim_client.py

```
@@ -175,7 +177,91 @@ def query_attitude(self):
+        background thread, until unsubscribe_state() is called. The
+        subscription is kept alive automatically.
+        """
+        if getattr(self, '_telemetry_thread', None):
+            print("[Wrapper] Already subscribed to telemetry.")
+            return
+        self._telemetry_running = True
+        self._telemetry_socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        self._telemetry_socket.settimeout(1.0)
+        self._telemetry_addr = (self.host, telemetry_port)
+
+        def _reader():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_reader`

Add return type annotation: `None`

(ANN202)